applications/asset_tracker_v2/src/modules/modem_module.c

```diff
@@ -61,6 +61,12 @@ static enum state_type {
 	STATE_SHUTDOWN,
 } state;
 
+/* Enumerator that specifies the data type that is sampled. */
+enum sample_type {
+	NEIGHBOR_CELL,
+	MODEM_DYNAMIC
```

Comment:
Could add MODEM_STATIC here to make the list complete even if it's not currently used

Reply:
Added MODEM_STATIC and BATTERY_VOLTAGE